tests/std/tests/P0645R10_text_formatting_args/test.cpp

```diff
@@ -229,11 +229,26 @@ void test_visit_monostate() {
     assert(visit_format_arg(visitor<Context>, basic_format_arg<Context>()) == Arg_type::none);
 }
 
+template <class Context, class FormatArgs>
+constexpr bool verify_lwg3810 = false;
+
+template <class Context>
+constexpr bool verify_lwg3810<Context, basic_format_args<Context>> = true;
+
+template <class Context>
+void test_lwg3810() {
+    auto args_store                         = make_format_args<Context>(1, 2, 3);
+    [[maybe_unused]] basic_format_args args = args_store;
+    static_assert(verify_lwg3810<Context, decltype(args)>);

```

Comment:
`static_assert(same_as<decltype(args), basic_format_args<Context>>);` isn't much longer, and would save the trouble of defining `verify_lwg3810`. `static_assert(same_as<decltype(basic_format_args(args_store)), basic_format_args<Context>>);` would save another line, and make it painfully obvious we're testing CTAD. (Pure style comment, ignore if you like.)